lib/irb/command/pushws.rb

```diff
@@ -15,7 +15,7 @@ class Workspaces < Base
       description "Show workspaces."
 
       def execute(*obj)
-        irb_context.workspaces.collect{|ws| ws.main}
+        irb_context.instance_variable_get(:@workspace_stack)[0..-2]&.collect{|ws| ws.main}
```

Comment:
Maybe we can remove `[0..-2]&.`. (And we don't need `&.` because `array[0..x]` never be nil.)

The command name `pushws` `popws` and the new message `Can't pop the last workspace on the stack` indicates that current workspace is in the last position of the workspace stack. I think we can simply show the stack here.

Also, showing all the stack makes it clear what `pushws` (noarg) is doing.
```

irb("a"):005> pushws "b"
=> [main, "a", "b"]
irb("b"):006> pushws
=> [main, "b", "a"] # "a" and "b" are visually swapped
irb("a"):007>
```